snuba/settings/__init__.py

```diff
@@ -133,6 +133,8 @@
 REPLACER_KEY_TTL = 12 * 60 * 60
 REPLACER_MAX_GROUP_IDS_TO_EXCLUDE = 256
 REPLACER_IMMEDIATE_OPTIMIZE = False
+REPLACER_PROCESSING_TIMEOUT_THRESHOLD = 2 * 60 * 1000  # 2 minutes in ms
+REPLACER_PROCESSING_TIMEOUT_THRESHOLD_KEY_TTL = 60 * 60  # 1 hour in s
```

Comment:
I don't disagree with it, but what is the negative result if we don't set a key TTL here. Just trying to understand the logic (and why we'd want it to be an hour)

Reply:
Just to prune old data since after a certain amount of time, that data isn't useful anymore cause a consumer would have picked up or passed the offset. I don't think the offset number itself ever resets to 0 but if for some reason it does,  having an offset existing in Redis would be really bad since it would skip messages till the number in Redis.

The TTL is an hour in case the consumer takes a long time to restart or if we're troubleshooting an issue.